galley/pkg/crd/validation/validation.go

```diff
@@ -161,6 +161,33 @@ func RunValidation(vc *WebhookParameters, kubeConfig string,
 	cmd.WaitSignal(stop)
 }
 
+// CleanupValidation delete validatingwebhookconfiguration if exists
+func CleanupValidation(stop <-chan struct{}, vc *WebhookParameters, kubeConfig string, probeOptions *probe.Options) {
+	clientset, err := kube.CreateClientset(kubeConfig, "")
+	if err != nil {
+		log.Fatalf("could not create k8s clientset: %v", err)
+	}
+	vc.Clientset = clientset
+
+	wh := &Webhook{
+		webhookName: vc.WebhookName,
+		clientset:   vc.Clientset,
+	}
+
+	for {
```

Comment:
Isn't this polling going to put _more_ load on the apiserver when validation is disabled vs. the existing code?

Reply:
In my understand, firstly, if the validation is false, the validation server should be disabled instead running with empty config. secondly, this cleanup logic only runs once when the pod passes the readiness.

Reply:
@ayj what is your conclusion?

Reply:
`CleanupValidation` doesn't exit until the stop channel closes. The code below will poll the apiserver once a second.

Reply:
`CleanupValidation` can run before the old pod (w/validation=true) exits. The old pod might still re-create the wrong webhook config.